kubernetes-client/src/main/java/io/fabric8/kubernetes/client/utils/ReflectUtils.java

```diff
@@ -39,6 +42,9 @@ public static ObjectMeta objectMetadata(Object obj) throws ReflectiveOperationEx
   }
 
   public static String namespace(Object obj) throws ReflectiveOperationException {
+    if( obj == null ) {
```

Comment:
I noticed ReflectUtils.namespace() wasn't getting used anywhere. Do we need this null check?

Reply:
Findbugs is asking for that actually.